src/carousel.scss

```diff
@@ -13,6 +14,15 @@ $button: #{$fd-namespace}-button;
   $fd-carousel-dot-size: 0.25rem !default;
   $fd-carousel-dot-active-size: 0.5rem !default;
 
+  @mixin after-position-overwrite() {
+    &::after {
```

Comment:
You can remove &::after from this mixin and place `@include after-position-overwrite` inside 
`@include buttonFocus() { @include after-position-overwrite();  }`